components/hab/src/command/plan/init.rs

```diff
@@ -83,25 +102,35 @@ pub fn start(
     }
 
     // We want to render the configured variables.
-    let rendered_plan = handlebars.template_render(PLAN_TEMPLATE, &data)?;
-    create_with_template(ui, &format!("{}/plan.sh", root), &rendered_plan)?;
-    ui.para(
+    if with_all || scaffold.is_none() {
+        let rendered_plan = try!(handlebars.template_render(FULL_PLAN_TEMPLATE, &data));
+        try!(create_with_template(
+            ui,
+            &format!("{}/plan.sh", root),
+            &rendered_plan,
+        ));
+    } else {
+        let rendered_plan = try!(handlebars.template_render(DEFAULT_PLAN_TEMPLATE, &data));
+        try!(create_with_template(
+            ui,
+            &format!("{}/plan.sh", root),
+            &rendered_plan,
+        ));
+    }
+    try!(ui.para(
         "The `plan.sh` is the foundation of your new habitat. You can \
-        define core metadata, dependencies, and tasks. More documentation here: \
-        https://www.habitat.sh/docs/reference/plan-syntax/",
-    )?;
-
-    let rendered_default_toml = handlebars.template_render(DEFAULT_TOML_TEMPLATE, &data)?;
-    create_with_template(
+        define core metadata, dependencies, and tasks.",
+    ));
+    let rendered_default_toml = try!(handlebars.template_render(DEFAULT_TOML_TEMPLATE, &data));
+    try!(create_with_template(
```

Comment:
We actually switched large portions of our codebase from `try!` to  the `?` syntax. They do the same thing but one is easier on our line length ;)